velox/common/caching/AsyncDataCache.h

```diff
@@ -427,6 +427,9 @@ class CoalescedLoad {
     setEndState(LoadState::kCancelled);
   }
 
+  /// Returns the cache space 'this' will occupy after loaded.
+  virtual int64_t size() const = 0;
```

Comment:
This causes a build failure in the tests because a derived class does not implement this method.

  267 | class TestingCoalescedSsdLoad : public TestingCoalescedLoad {
      |       ^~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/localuser/workspace/./velox/common/caching/SsdFile.h:19,
                 from /home/localuser/workspace/./velox/common/caching/SsdCache.h:19,
                 from /home/localuser/workspace/velox/common/caching/tests/AsyncDataCacheTest.cpp:18:
/home/localuser/workspace/./velox/common/caching/AsyncDataCache.h:431:19: note:     'virtual int64_t facebook::velox::cache::CoalescedLoad::size() const'
  431 |   virtual int64_t size() const = 0;
      |                   ^~~~
[